src/runtime.sh

```diff
@@ -515,7 +515,11 @@ runtime::list() {
     # Look for all the pids associated with any of the rootfs.
     for pid in $(lsns -n -r -t mnt -o pid); do
         if [ -e "/proc/${pid}/root/${lock_file}" ]; then
-            name=$(awk '($5 == "/"){print $4; exit}' "/proc/${pid}/mountinfo" 2> /dev/null)
+            name=$(awk '($5 == "/"){print $4; exit}' "/proc/${pid}/mountinfo" 2> /dev/null || echo "<gone>")
+            if [ "${name}" = "<gone>" ]; then
+                continue
+            fi
+
```

Comment:
Are you trying to catch the case when you can't find the root mount, or when awk fails to read the file? Looks like the latter?

Reply:
When awk fails to find the file because the PID terminated. This causes `enroot list -f` to exit with error code `2`.